src/appleseed/CMakeLists.txt

```diff
@@ -989,6 +989,13 @@ set (renderer_kernel_intersection_sources
     renderer/kernel/intersection/triangletree.h
     renderer/kernel/intersection/trianglevertexinfo.h
 )
+
+if (WITH_EMBREE)
+    list (APPEND renderer_kernel_intersection_sources
+        renderer/kernel/intersection/embreescene.cpp
+        renderer/kernel/intersection/embreescene.h)
+endif()
```

Comment:
Insert space after `endif` (in our CMakeLists.txt files all CMake keywords are followed by a space).